README.md

```diff
@@ -21,27 +28,34 @@ const RootStack = createStackNavigator<RootStackParamList>();
   <RootStack.Screen
     name="Profile"
     component={Profile}
-    initialParams={{ userId: user.id }}
+    initialParams={{ userId: user.id, customTypeParam: new MyCustomType()}}
```

Comment:
Can we change this with a real type example? I don't often see `new` being used with types.

Perhaps a User interface with a name key?

Reply:
I blame copilot

Reply:
Oh actually these are weird -- I will give them a closer read.